variety/PrivacyNoticeDialog.py

```diff
@@ -0,0 +1,51 @@
+# -*- Mode: Python; coding: utf-8; indent-tabs-mode: nil; tab-width: 4 -*-
+### BEGIN LICENSE
+# Copyright (c) 2012, Peter Levi <peterlevi@peterlevi.com>
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 3, as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranties of
+# MERCHANTABILITY, SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR
+# PURPOSE.  See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program.  If not, see <http://www.gnu.org/licenses/>.
+### END LICENSE
+
+from gi.repository import Gtk # pylint: disable=E0611
+
+from variety_lib.helpers import get_builder
+
+
+class PrivacyNoticeDialog(Gtk.Dialog):
+    __gtype_name__ = "PrivacyNoticeDialog"
+
+    def __new__(cls):
```

Comment:
Looking at this code - it's again some legacy boilerplate code from whatever Quickly generated in the very beginning and I haven't come to refactoring it. No need to do anything in this PR, but we shouldn't be repeating these same 10-12 lines in every dialog we have. For most of those I'm not sure we even need the dedicated class, a simple factory function that takes the name (e.g. 'PrivacyNoticeDialog' here) and initializes and returns new_object would do.